src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/intl-duration.ts

```diff
@@ -0,0 +1,49 @@
+const __ = require('oro/translator');
+
+type IntlDuration = {
+  translationKey: string;
+  value: number;
+}
+
+const translateDuration = (...args: IntlDuration[]): string => {
+  let values = args.filter((duration: IntlDuration) => duration.value > 0);
+
+  if (values.length === 0 && args.length > 0) {
+    values = [args[args.length - 1]];
+  }
+
+  return values
+    .map((duration: IntlDuration) => __(duration.translationKey, {count: duration.value}, duration.value))
+    .join(' ');
+};
+
+export const formatSecondsIntl = (duration: number): string => {
+  const days = Math.floor(duration / 86400);
```

Comment:
maybe overkill but could be more clear IMHO to do:
```suggestion
  const days = Math.floor(duration / (60 * 60 * 24);
```

Reply:
I think that `days = floor(duration / 86400)` is already quit clear. I'm not convinced it make it clearer to use 60*60*24 but another opinion is welcome.

@ValentinMumble, what do you think ?

Reply:
I'd put `3600 * 24`

Reply:
:trollface:

Reply:
(don't care, I know 86400 is the number of seconds in a day, either way is clear to me)